bundles/org.openhab.binding.qbus/README.md

```diff
@@ -0,0 +1,114 @@
+- `thermostats`: Thermostats - normal and PID
+- `scene`: Scenes
+- `co2`: CO2 
+- `rollershutter`: Rollershutter 
+- `rollershutter_slats`: Rollerhutter with slats
+
+For now the following Qbus things are not yet supported but will come:
+
+- DMX
+- Timer 4 & 5
+- HVAC
+- Humidity
+- Renson
+- Duco
+- Kinetura
+- Energy monitor
+- Weather station
+
+
+## Discovery
+
+The discovery service is not yet implemented but the System Manager III software of Qbus generates things and item files from the programming, which you can use directly in openHAB.
+
+## Thing Configuration
+
+This is an example of the things configuration file:
+
+```
+Bridge qbus:bridge:CTD001122 [ addr="localhost", sn="001122", port=8447, refresh=10 ] {
+    dimmer                   1     "ToonzaalLED"      [ dimmerId=100 ]
+    onOff                    30    "Toonzaal230V"     [ bistabielId=76 ]
+    thermostat               50    "Service"          [ thermostatId=99 ]
+    scene                    70    "Disco"            [ sceneId=36 ]
+    co2                      100   "Productie"        [ co2Id=26 ]
+    rollershutter            120   "Roller1"          [ rolId=268 ]
+    rollershutter_slats      121   "Roller2"          [ rolId=264 ]
+}
+```
+
+
+| Property   | Default  | Required | Description    |
+|------------|----------|----------|----------------|
+| `addr`    | localhost | YES      | The ip address of the machine where the Qbus Server runs |
+| `sn`      |           | YES      | The serial number of your controller |
+| `port`    | 8447      | YES      | The communication port of the client/server |
+| `refresh` | 5         | NO       | Refresh time - After x minutes there will be a check if server is still running and if client is still connected. If not - reconnect |
+
+## Channels
+
+| Thing Type ID  | Channel Name         | Read only  | description                                             |
+|---------------------|---------------|---------------------------------------------------------|
+| `onOff`               | switch        | No | This is the channel for Bistable, Timers and Intervals  |
+| `dimmer`              | brightness    | No | This is the channel for Dimmers 1&2 buttons and CLC     |
+| `scene`               | Switch        | No | This is the channel for scenes                          |
+| `co2`                 | co2           | Yes | This is the channel for CO2 sensors                     |
+| `rollershutter`       | rollershutter | No | This is the channel for rollershutters                  | 
+| `rollershutter_slats` | rollershutter | No | This is the channel for rollershutters with slats       |
+| `thermostat`          | setpoint      | No | This is the channel for thermostats setpoint            |
+| `thermostat`          | measured      | Yes | This is the channel for thermostats currenttemp         |
+| `thermostat`          | mode          | No | This is the channel for thermostats mode                |
```

Comment:
Are these Channels or Things? Either the chapter name, the column names or the description doesn't fit.